doc/sphinx-guides/source/installation/config.rst

```diff
@@ -178,56 +178,383 @@ Persistent Identifiers and Publishing Datasets
+**DataCite** (https://www.datacite.org) is the recommended DOI provider
 (see https://dataversecommunity.global for more on joining DataCite through the Global Dataverse Community Consortium) but **EZID**
 (http://ezid.cdlib.org) is an option for the University of California according to
 https://www.cdlib.org/cdlinfo/2017/08/04/ezid-doi-service-is-evolving/ .
 **Handle.Net** (https://www.handle.net) is the HDL provider.
 
-Once you have your DOI or Handle account credentials and a namespace, configure your Dataverse installation
-using the JVM options and database settings below.
+Once you have your DOI or Handle account credentials and a prefix, configure your Dataverse installation
+using the settings below.
+
+ 
+Configuring PID Providers
++++++++++++++++++++++++++
+
+There are two required global settings to configure PID providers - the list of ids of providers and which one of those should be the default.
+Per-provider settings are also required - some that are common to all types and some type specific. All of these settings are defined 
+to be compatible with the MicroProfile specification which means that
+
+1. Any of these settings can be set via system properties (see :ref:`jvm-options` for how to do this), environment variables, or other
+   MicroProfile Config mechanisms supported by the app server.
+   `See Payara docs for supported sources <https://docs.payara.fish/community/docs/documentation/microprofile/config/README.html#config-sources>`_.
+2. Remember to protect your secrets. For passwords, use an environment variable (bare minimum), a password alias named the same
+   as the key (OK) or use the `"dir config source" of Payara <https://docs.payara.fish/community/docs/documentation/microprofile/config/directory.html>`_ (best).
+
+   Alias creation example:
+
+   .. code-block:: shell
+
+      echo "AS_ADMIN_ALIASPASSWORD=changeme" > /tmp/p.txt
+      asadmin create-password-alias --passwordfile /tmp/p.txt dataverse.pid.datacite1.datacite.password
+      rm /tmp/p.txt
+
+3. Environment variables follow the key, replacing any dot, colon, dash, etc. into an underscore "_" and all uppercase
+   letters. Example: ``dataverse.pid.default-provider`` -> ``DATAVERSE_PID_DEFAULT_PROVIDER``
+   
+Global Settings
+^^^^^^^^^^^^^^^
+
+The following three global settings are required to configure PID Providers in the Dataverse software:
+
+.. _dataverse.pid.providers:
+
+dataverse.pid.providers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A comma-separated list of the ids of the PID providers to use. IDs should be simple unique text strings, e.g. datacite1, perma1, etc.
+IDs are used to scope the provider-specific settings but are not directly visible to users. 
+
+.. _dataverse.pid.default-provider:
+
+dataverse.pid.default-provider
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ID of the default PID provider to use.
+
+.. _dataverse.pid.provider-implementations-directory:
+
+dataverse.pid.provider-implementations-directory
```

Comment:
**issue**: This seems to be outdated, now that it has been moved to the `dataverse.spi.pidproviders.directory` setting.

**suggestion**: Replace with new name.

Reply:
done